src/drivers/bitbucket_cloud.js

```diff
@@ -148,6 +148,10 @@ class BitBucketCloud {
     });
   }
 
+  async pipelineRestart(opts = {}) {
+    throw new Error('BitBucket Cloud does not support workflowRestart!');
```

Comment:
This would probably require a mention [here](https://github.com/iterative/cml/wiki/Backend-Supported-Features), right?

Reply:
Not really, the only mention is that BB does not have runners!

Reply:
🤦🏼 👌🏼